### DISCUSSION TITLE
Mechanic: NWO Action

### DISCUSSION THREAD

**Discussion by RebelusQuo** [RebelusQuo/the-wall-is-down#14] [Ideas]
The rules document states:
> 6.2 Action phases
> .. , each player plays one card from her hand for one of these effects:
> - ..
> - Alter the NWO track

> 7. New world order (NWO)
> .. A player can, as send a card to any slot of the NWO to put her supremacy token on that slot if it is empty, or remove the token of an opposing player if not.

This sounds like it's a variation of the Space Race track in TS, so is the assumption that you can normally only perform this action once per turn correct in your game as well? Understanding the utility of the NWO track related cards very much hinges on understanding this base game rule.

**Comment by rodrigoSantamaria** [comment]
Yes, it's similar to the space race, but I left open to be able to do it more than once in a turn. 
As it is designed, it is usually a move much less powerful than playing ops/events, so in playtests it did not feel as something worth being restricted.

**Comment by RebelusQuo** [comment]
Alright, that's quite different. Perhaps worth emphasizing in the rulebook?

**Reply by rodrigoSantamaria** [reply]
Yes, maybe. In the rulebook it says nothing about limits, but it could be good to stress that you can use this ops an unlimited number of times.
